RecoHI/HiEgammaAlgos/python/RecoHiEgamma_EventContent_cff.py

```diff
@@ -24,7 +26,9 @@
     "drop recoPFClusters_*_*_*",
     "keep recoElectronSeeds_*_*_*",
     "keep recoGsfElectrons_*_*_*",
-    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*'
+    'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducer_*_*',
+    'keep recoPhotons_gedPhotonsTmp_*_*',
```

Comment:
is relinking step added as well (the one that makes gedPhotons post-PF in pp reco)?
It's unfortunate to have these intermediate names in the HI output, coming with *Tmp